package.json

```diff
@@ -65,6 +65,7 @@
 		"debug-brk": "meteor run --inspect-brk",
 		"lint": "meteor npm run stylelint && meteor npm run jslint",
 		"jslint": "eslint .",
+		"jslint-fix": "eslint . --fix",
```

Comment:
```suggestion
```

Probably ok to add.  But kinda outside of this scope and probably requires its own conversation

Reply:
agreed.. you don't need a special script entry for this.. you can do `meteor npm run eslint -- --fix` which does the same